docs/references/sql-introspection.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 3
+description: "API reference for inspecting the invocation status and service state."
+---
+# SQL Introspection API
+
+This page contains the reference of the introspection tables.
+To learn how to access the instrospection interface, check out the [instrospection documentation](/operate/introspection).
+
+## Table: `state`
+
+| Column name | Type | Description |
+|-------------|------|-------------|
+| `partition_key` | `UInt64` | Internal column that is used for partitioning the services invocations. Can be ignored. |
+| `service_name` | `Utf8` | The name of the invoked service. |
+| `service_key` | `Utf8` | The key of the Virtual Object. |
+| `key` | `Utf8` | The `utf8` state key. |
+| `value_utf8` | `Utf8` | Only contains meaningful values when a service stores state as `utf8`. This is the case for TypeScript services since the TypeScript SDK serializes values as JSON. |
+| `value` | `Binary` | A binary, uninterpreted representation of the value. You can use the more specific column `value_utf8` if the value is a string. |
+
+## Table: `sys_invocation`
+
+| Column name | Type | Description |
+|-------------|------|-------------|
+| `id` | `Utf8` | [Invocation ID](/operate/invocation#invocation-identifier). |
+| `target` | `Utf8` | Invocation Target. Format for plain services: `ServiceName/HandlerName`, e.g. `Greeter/greet`. Format for Virtual Objects/Workflows: `VirtualObjectName/Key/HandlerName`, e.g. `Greeter/Francesco/greet`. |
+| `target_service_name` | `Utf8` | The name for the invoked service. |
+| `target_service_key` | `Utf8` | The key of the Virtual Object or of the Workflow. Null for regular services. |
+| `target_handler_name` | `Utf8` | The invoked handler. |
+| `target_service_ty` | `Utf8` | The service type. Either `service` or `virtual_object` or `workflow`. |
+| `invoked_by` | `Utf8` | Either `ingress` if the service was invoked externally or `service` if the service was invoked by another Restate service. |
+| `invoked_by_service_name` | `Utf8` | The name of the invoking service. Or `null` if invoked externally. |
+| `invoked_by_id` | `Utf8` | The caller [Invocation ID](/operate/invocation#invocation-identifier) if the service was invoked by another Restate service. Or `null` if invoked externally. |
+| `invoked_by_target` | `Utf8` | The caller invocation target if the service was invoked by another Restate service. Or `null` if invoked externally. |
+| `pinned_deployment_id` | `Utf8` | The opaque service deployment ID that has been committed for this invocation; this is set after the first journal entry is stored for this invocation. |
```

Comment:
Can we make the language a bit easier for this? 
`The ID of the service deployment that started processing this invocation, and will continue to do so (e.g. for retries). This gets set after the first journal entry has been stored for this invocation.`

Reply:
👍